packages/front-end/components/FlagCard/FlagCard.tsx

```diff
@@ -218,6 +217,26 @@ export default function FlagCard({
             }
           />
         ) : null}
+
+        {!data?.isGuardrail && data.rowResults.suspiciousChange ? (
+          <CardItem
+            label="Suspicious"
+            tooltip={data.rowResults.suspiciousChangeReason}
+            value={<span style={{ color: "var(--pink-a11)" }}>suspicious</span>}
+          />
+        ) : null}
+
+        {data.rowResults.guardrailWarning ? (
```

Comment:
Where did this come from? It seems like previously this was only used for Safe Rollouts; did we want to highlight non-significant negative guardrail trends here?

It already will probably have a risk warning, and I don't know if it adds a ton. Kind of nbd either way, just wondering if it's additive.

<img width="436" height="470" alt="Screenshot 2025-08-13 at 9 44 28 AM" src="https://github.com/user-attachments/assets/6fd332dd-1d64-44b6-8a2d-c806a129f8a8" />

I think something like this makes sense if we continue with the overhaul of how guardrails work in an experiment and make them one-sided or something.